### PR TITLE
Corrected typo mistakes in pattern-library files.

### PR DESCRIPTION
**Pull request by sanganinamrata**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [x] Bug fix or improvement
 

## Summary
Corrected typo mistakes in **pattern-library files** for version 2.x .

### CODE REVIEW

File: guides/v2.0/pattern-library/displaying-data/datatable/datatable.md [hunk 1]
```diff
@@ -469,7 +469,8 @@ Some data-tables will be used to display "settings" content. In these instances
...
+
+The data table can also accommodate buttons in rows, and has a special case in the "Single Select" button.  This button functions not unlike a typical radio button control, but is used instead of the radio button when the task flow or experience in the {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %} application are better served with the more implicit action of selecting a single choice from a data table. This provides greater visibility and immediate confirmation of the action to the user. The use of this should be carefully considered in the context of the experience to determine if it provides greater understanding to the user than radio buttons.
```
Comment (shrielenee) [inline guides/v2.0/pattern-library/displaying-data/datatable/datatable.md]: Remove double spaces before new sentences.

File: guides/v2.1/pattern-library/displaying-data/datatable/datatable.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ Views can be named or deleted by clicking an edit icon. Refer to "Adding a custo
 ## Filters {#filters}
 Filters allow users to customize the display of information in an associated data-table. By entering desired parameters (for example, a date range) in the available filter input sections table returns a modified view, making visible only the records meeting the user-determined criteria.
 
-### Advanced Filters
+<h3>Advanced Filters</h3>
 To access the Advanced Filters the user may click/tap the "Filters" button found in the table controls area. When activated, the button transforms into a tab and displays the available filters in a 'drawer' which opens between the table controls and the table data. The available filters are determined by the columns present in the table; therefore, if the user customizes the column visibility of the table to show some columns and hide others, the corresponding filters for these columns will display or hide in the filter 'drawer'.
```
Comment (shrielenee) [inline guides/v2.1/pattern-library/displaying-data/datatable/datatable.md]: Revert this change. We use markdown for headings, not HTML.

File: guides/v2.1/pattern-library/displaying-data/datatable/datatable.md [hunk 3]
```diff
@@ -471,8 +471,9 @@ Some data-tables will be used to display "settings" content. In these instances
...
+The data table can also accommodate buttons in rows, and has a special case in the "Single Select" button.  This button functions not unlike a typical radio button control, but is used instead of the radio button when the task flow or experience in the {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %} application are better served with the more implicit action of selecting a single choice from a data table. This provides greater visibility and immediate confirmation of the action to the user. The use of this should be carefully considered in the context of the experience to determine if it provides greater understanding to the user than radio buttons.
```
Comment (shrielenee) [inline guides/v2.1/pattern-library/displaying-data/datatable/datatable.md]: Remove double spaces before sentences.

File: guides/v2.0/pattern-library/displaying-data/datatable/datatable.md (hunk 1 above)
Comment (sanganinamrata) [reply guides/v2.0/pattern-library/displaying-data/datatable/datatable.md]: @shrielenee Done!

File: guides/v2.1/pattern-library/displaying-data/datatable/datatable.md (hunk 2 above)
Comment (sanganinamrata) [reply guides/v2.1/pattern-library/displaying-data/datatable/datatable.md]: Done!

File: guides/v2.1/pattern-library/displaying-data/datatable/datatable.md (hunk 3 above)
Comment (sanganinamrata) [reply guides/v2.1/pattern-library/displaying-data/datatable/datatable.md]: Done

**Comment by sanganinamrata** [comment]
@shrielenee , I'm done with the mentioned changes.

**Comment by shrielenee** [comment]
Great @sanganinamrata!
